include/libdnf5/base/goal_elements.hpp

```diff
@@ -137,12 +140,22 @@ enum class GoalAction {
     REASON_CHANGE,
     ENABLE,
     DISABLE,
-    RESET
+    RESET,
+    REPLAY_INSTALL,
+    REPLAY_REMOVE,
+    REPLAY_UPGRADE,
+    REPLAY_REINSTALL,
+    REPLAY_REASON_CHANGE,
+    REPLAY_REASON_OVERRIDE,
+    REVERT_COMPS_UPGRADE
```

Comment:
Just a question - why the last item does not use replay prefix?

Reply:
My thinking was that since it happens during the reverting the `REVERT` prefix is more appropriate. Technically it has nothing to do with replay even though the result of the revert is used by replay.

Reply:
Thank you